addons/event/models/event_registration.py

```diff
@@ -34,7 +34,7 @@ def _get_random_barcode(self):
     event_id = fields.Many2one(
```

Comment:
Commit message explains what it does but not really why, and it seems also lacking a few of the changes done here.

For example "Remove the skip button in the registration editor form" could use some context to explain why we don't want it anymore.